lib/eslint-patch.js

```diff
@@ -1,13 +1,22 @@
 "use strict";
 
 const Plugins = require("eslint/lib/config/plugins");
+const ModuleResolver = require("eslint/lib/util/module-resolver");
+
+const ConfigFile = require("eslint/lib/config/config-file");
 
 const Config = require("eslint/lib/config");
 const ConfigUpgrader = require("./config_upgrader");
 
-const docs = require("./docs");
-
-module.exports = function patch(eslint) {
+module.exports = function patch() {
+  const resolve = ModuleResolver.prototype.resolve;
+  ModuleResolver.prototype.resolve = function(name, path) {
+    try {
+      return resolve.apply(this, [name, path]);
+    } catch(e) {
```

Comment:
I think using the same message sounds fine. Are you saying it's already being printed? I didn't see it when doing local QA, so I thought it wasn't included there